crates/aptos/src/move_tool/mod.rs

```diff
@@ -524,6 +506,17 @@ pub struct TestPackage {
     pub dump_state: bool,
 }
 
+fn fix_bytecode_version(
+    bytecode_version_in: Option<u32>,
+    language_version: Option<LanguageVersion>,
+) -> Option<u32> {
+    if let Some(language_version) = language_version {
+        Some(language_version.infer_bytecode_version(bytecode_version_in))
+    } else {
+        bytecode_version_in
+    }
+}
```

Comment:
So, does this override the byte code version if it's move 2?

What does "fix" mean?

Reply:
Yes.  It overrides the selected bytecode version if language_version = 2.

Reply:
We should never override a direct flag given by the user. AFAICT the previous logic already gives you an error if you select an incompatible bytecode version. The inferred bytecode version should be used if the user hasn't specified anything.

Reply:
I updated `fix_bytecode_version` to leave the input value if it's not `None`.